changelogs/fragments/package_facts_fix.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - package_facts module when using 'auto' will return the first package manager found that provides an output, instead of just the first one, as this can be forgein and not have any packages.
```

Comment:
```suggestion
  - package_facts module when using 'auto' will return the first package manager found that provides an output, instead of just the first one, as this can be foreign and not have any packages.
```